### DISCUSSION TITLE
If negotiated rate is based on percentage of charge--then how would this be reported in the in-network file?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#23] [Q&A] [answered]
Since the rate must be published in a dollar amount, how should items/services be represented in the in-network file if based on the percentage of charge?    Would the negotiation type be 'derived'?   How would you report the negotiated rate in the file, since it varies on charge for the item/service?  Or do these items/services get reported?

**Comment by Kam-cog** [comment]
Here is some explanation about the various types of reimbursements and how it should be reported  that I see in the final rules PDF.
Page 259.
By contrast, under a reference-based pricing model with a defined network, payers have 
contractual agreements to reimburse providers based on a percentage of a different rate that is 
known or determinable by the parties (usually 120 percent to 200 percent of the Medicare rate), 
which is subject to change based upon adjustments that can be specific to the participant, 
beneficiary, or enrollee, such as age, gender, and severity of illness.  To represent this type of 
               
arrangement, and other provider reimbursement models that are based upon participant, 
beneficiary, or enrollee-specific adjustments, the final rules clarify that plans and issuers are 
required to include for each item or service in the In-network Rate File, the base negotiated rate 
that applies before adjusting for participant, beneficiary, or enrollee -specific characteristics.  
The negotiated rate in the referenced-based pricing model must be represented as a dollar value 
that is the result of the calculation of the referenced amount and the applicable reference-based 
percentage.  For example, a plan calculates provider reimbursement using a reference-based 
pricing model that sets reimbursement to Provider X at 120 percent of the Medicare rate for 
covered Item A. The reference-based percentage used to determine the base negotiated rate 
would be 120 percent.  In the general course of business, the plan determines the Medicare rate 
for Item A using participant, beneficiary, or enrollee-specific characteristics, but, because there is 
no specific participant, beneficiary, or enrollee for purposes of populating the In-network Rate 
File, the plan or issuer must report the base negotiated rate that would apply prior to application 
of any participant, beneficiary, or enrollee-specific characteristics. In this example, the Medicare 
rate for Item A is $150, before applying adjusters for participant, beneficiary, or enrollee-specific 
characteristics.  Therefore, the plan would report a negotiated rate for Item A when received 
from Provider X of $180 ($150 multiplied by 120 percent) and must include this rate in the In
network Rate File.

**Reply by tdfow** [reply]
Seems the schema doesn't support identifying the 'basis' for calculation.  There is a reference in the rule, where it indicated notes could be added to clarify how the amount was derived  yet the schema doesn't seem to support anything like that--so how would the 'basis' be known for the reporting scenarios if based on a percentage?

**Reply by iGokul-G** [reply]
Totally agree. This would also help differentiate  payment methodologies  like Perdiem vs case rate etc.

**Reply by tdfow** [reply]
Shouldn't there be a 'notation' type field in the schema to denote, the rate is based on a percentage of charge and derive that the rate is based on a given 'basis' for reporting purposes, such as the rate is based on percentage of x dollars and the x dollars will vary the rate--but the negotiated rate published is using x dollars, where x = 100.00 for example.  Otherwise,  how do we report the rates that are based on percentage of charge?

**Reply by transparency123** [reply]
Has there been any update from CMS on how to deal with percent of charge  reimbursements using the published schema given payers do NOT have visibility (not always unless hospitals publish their chargemasters) into billed charges for "all" covered items and services?

**Reply by rpackar** [reply]
Same question here as well. Any thoughts would be appreciated. Is anyone considering using an ASP typoe algorithm, where you would apply the percent of charge to an average sales price, then reflect that in the "basis for calculation"

**Reply by ghost** [reply]
We are mining as much as possible our enterprise data warehouse for any historical claim related/charge information we can use for cases we have a billing code like this. This just adds another layer of effort and complexity to generate this file on a monthly basis to provide the required value as dollar amounts for this schema field. Unlike the allowed-amount file there has been no guidance/criteria to base how this historical information should be calculated.

**Reply by shaselton-usds** [reply]
Thanks for pulling the text @Kam-cog. 

To provide context, we initially thought through possible implementation scenarios like this:

```
{  
  "reporting_entity_name": "medicare",  
  "reporting_entity_type": "medicare",  
  ...
  "in_network": [{     
    "negotiation_arrangement": "reference based",    
    "name": "Knee Replacement",    
    ...
    "negotiated_rates": [{      
       "reference_based_percentage": "103",
       "reference_based_standard": "medicare",
       "weighted_average_price": 1000.00, // drg base rate = (location, wage index, hospital type, etc). Average location variable number, Average wage index number, Average hospital type number
     ....
}
```
It added more optional attributes which drove up complexity.

The core question from @tdfow, it looks like we might need to technically add `referenced based` to one of the possible values for `negotiation_arrangement` to signify the arrangement, yes? If so, the documentation will be added to include `referenced based` (it might have been an oversight with the initial publication).

@transparency123, @rpackar, @BobSyracuse -- this goes a little beyond the initial question, but this is a good point and something that we were thinking about with the `weighted_average_price`.  Would it be possible to create a new discussion to continue that conversation there to track progress?

**Reply by MMath19103** [reply]
@shaselton-usds would the amount be allowed to be a percentage instead of a whole dollar amount?

**Reply by shaselton-usds** [reply]
@MMath19103 No, a dollar amount is required.

**Reply by ghost** [reply]
@shaselton-usds It will add out a new discussion on the information you provided.

**Reply by iGokul-G** [reply]
@shaselton-usds 

Adding a new “negotiation_arrangement” value to represent commercial pricing based on Medicare/CMS  Pricers  would help . But payers do have commercial pricing  payment agreements based off straight percentage of CMS  Pricer’s output   like   
Percentage (%) of Medicare APC   , 
Percentage (%) of Medicare ASC , 
Percentage (%) of Medicare IPPS and potentially in future could use  percentage of other CMS Pricer like SNF, IRF , IPF etc. 

In the above  scenarios do we really need to represent  data attribute listed as  "weighted_average_price"  line mandatoryinstead why cant we just representing the percentage ( % ) and may be a new field for CMS Pricer type as listed below 

  "in_network": [{     
    "negotiation_arrangement": "reference based",    
    "name": "Knee Replacement",    
    ...
    "negotiated_rates": [{      
       "reference_based_percentage": "103",
       "reference_based_standard": "medicare",
       "reference_based_type": "APC",


Payers could also use CMS Pricer data attributes like APC number , OPPS payment status indicators  ( SI ) etc.  to drive the payment which would need additional allowed values with in billing_code_type field

**Reply by Ginger228** [reply]
The guidance is using the % of a medicare rate.  We are looking for guidance with the negotiated rate is the % of billed charges.  This is typically with DME providers.

**Reply by tdfow** [reply]
Could negotiation_arrangement be expanded to possibly address per diem ("perdiem") or percentage of charge ("percentageofcharge"), where then the negotiated_rate could be published to indicate the perdiem rate or the percentage---since we don't have a claim to indicate the charge amount or the number of days, etc.---then we could publish the 'rate' based on this type of arrangement?

**Reply by Ginger228** [reply]
Wondering if we could get a response for this item.  There are contracts that reimburse the provider/facility at % of the billed charge or invoice.  If the bill hasn't been received how should the payer derived the $ amount for service not yet performed?  @shaselton-usds

**Reply by tdfow** [reply]
Will there be changes to schema to address additional negotiation_arrangement or some type of notation to denote the percentage of charge reporting?  @shaselton-usds

**Reply by tdfow** [reply]
@shaselton Any updates on whether there will be other values for 'negotiation_arrangement' or will there be any 'referenced based' fields updated on the negotiated price object to denote, percentage of charge scenarios or per diem related rates?

**Comment by clovis517** [comment]
This is a significant concern.  Why does the final rule imply that this situation only exists for out-of network? _"Given that some plans and issuers determine the allowed amount for certain out-of network items and services based on a percentage of billed charges, the final rules provide that a percentage can be disclosed instead of a dollar amount, if plans and issuers reimburse out-of network providers a percentage of the billed charges for a covered item or  service."_  The irony is we don't need this for out of network since we can use claims history instead.  How can the negotiated price object be populated with the "_type of negotiated rate as well as the dollar amount of the negotiated rate_" if the negotiated rate is a percentage, to be applied to an unknown amount?  Harvesting the machine readable hospital charge master files is not a solution, there is no common data structure and the data is incomplete at more than half of the hospitals we examined.  Even if we solve that we are faced with a unit of measure question.  Negotiated Dollar Amount per HCPCS service or per HCPCS Unit; per Admission or per Day?  Won't any real-world use-case of this data require a unit of measure attribute in the Negotiated Price Object?

**Reply by rakeshkatte** [reply]
Another variant of this problem is when the full charges are allowed for some services. There is no way to represent this in the schema. If the percentage option is allowed, then we can report the price as 100% of charges. 
And yes, even we have the situation of prices being defined per unit, per Diem or in case of anesthesia per 10min (low) or 15min (high) units.

**Comment by ryan-thomas-peterson** [comment]
In the absence of any update and given that many of our methodologies rely on underlying charge structures (for things like inlier calculations, outlier attachment points and where reimbursement falls to a percentage charge), we're considering adding the assumed charge to the 'description' field of the 'In-Network' Object.   This at least grounds the result in what the underlying assumptions was since many services (especially grouped ones like DRGs and APCs) can have wild variation in what the underlying charges or length of stay is for the same provider depending on other factors.

How have others approached this?

**Reply by ericsilvertx** [reply]
My situation is somewhat similar for our In Network File.  In most of our instance the in patient provider submit an initial demand rate, it gets into the claim line item then it gets "adjusted" based on a certain percentage off from the initial demand rate.  has anyone else have that same challenge of having a payer configuration where there is not a clearly defined fee schedule?

**Comment by ryan-thomas-peterson** [comment]
@shaselton-usds,  To support this use case, could the schema support an optional 'presumed provider charge' field as part of the negotiated rate object?  Contracted rates are well-defined (or at least better defined) when the underlying charge is known for the service but as others have pointed out they may not always know the underlying charge.

**Comment by Ginger228** [comment]
@shaselton-usds
Any updates on the posted of the payment method = % of billed charges?  The discussion propose using the average amount paid from claim history of the item/service but this wouldn't be accurate. 

A recommendation is to add another value to be used with the negotiated type. 

"negotiated_prices":[{
"negotiated_type" : "percent_charge",
"negotiated_rate" :  75,
"expiration_date" : "2022-12-31",
"service_code" : [ "10"],
"billing_class" : "professional"

**Reply by tulsin108** [reply]
I completely agree @Ginger228 - this would be the ideal solution for this problem. We cannot control what providers bill, so when our agreement with them is to pay a percentage of billed charges, without a claim to refer to - we cannot product a dollar amount for the In Network Rates file. I think your suggestion is the best solution to this problem. If we could provide what the contracted percent of billed rate is, then a consumer could potentially reach out to the provider to ask what they would normally bill the insurer for the service. Hospitals are supposed to be publishing their charges anyways per the Hospital Price Transparency mandate. 

Also, as you stated, the idea of using an average paid amount from claims history would not be accurate and would therefore be misleading to consumers, which is the opposite of the intent of this mandate. Providers can bill whatever they want for any given service, so to come up with an average billed amount for a service would not only add complexity to the technical solution, but would not be accurate, as you stated. 

Without a solution to this problem, I am afraid many plans will be non-compliant with the MRF mandate when it comes to reporting dollar amounts for percentage of billed negotiated rates. 

@shaselton-usds could we please get further guidance from CMS on this issue? Could this issue please be escalated and addressed soon? 

I know that many plans whom are part of the Blues Association or who are using vendors need to start finalizing technical solutions and begin generating test files well before July 1st.

**Comment by Labouv0506** [comment]
This solution would be helpful for us as well. Many of our in network contracts list an "all others or default" term as a % of billed charge. 
Basically if a procedure is not covered under the contract specifically, it is caught in a final term that is defined as a % of the billed charge, 
Currently we are unsure how to display this without the ability to show the % of chg on the in network allowable file also.

**Reply by rgouldkc** [reply]
We have a similar situation where if there is not a specific contracted rate for a billing code a % discount off billed charges is used. Additionally we have fee schedules where the contracted rate is a percent of the billed charge. So this would be very helpful for us as well. For consistency this should be reflected as a percentage of the billed charge in all cases to avoid confusion between percentage of charge vs percentage discount off billed charge such as our default percentage discount since that can also be expressed as a percentage of charge.

**Comment by angelamarie0975** [comment]
@shaselton-usds Has there been any more discussions on this topic at CMS? Since the billed charge is not known at the provider contract level - we just have the contract states just a % rate, what is the expectation of the insurers to get the billed charge? There are discussions about different approaches being used to determine the billed amount, but I don't know if there has been anything defined as to how we should be deriving the billed charge. Is there any guidance for this from CMS?

**Comment by shaselton-usds** [comment, marked as answer]
Closing out this old thread. Percentage of billed charges is now supported by the v1.0 schema.
